react/react1/week2/react1-week2/src/Main.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { useState } from "react";
+import TodoHeader from "./components/TodoHeader";
+import TodoList from "./components/TodoList";
+import './TodoListStyle.css'
+
+const todoArray = [
+    {
+      id: 1,
+      description: "Get out of bed",
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+    },
+  ];
+
+
+
+
+function Main(){
+    const [todos, setTodos]=useState(todoArray)
+    const buttonClick =()=>{
+        const todoArrayLen =todos.length
+        const addTodoList=[
```

Comment:
take care with the naming, you're adding an Item, not a list
```suggestion
        const addTodoItem=[
```